### DISCUSSION TITLE
Disable ellipsis (…) unless snippets are expanded

### DISCUSSION THREAD

**Discussion by TheZoq2** [autozimu/LanguageClient-neovim#1185] [Q&A]
I'm aware that this might not be the correct place to ask this, but I'm also not sure if the source of my problem is `rust-analyzer`, `LanguageClient-neovim` or `ncm2`

My problem is best described by a video. Here I want to complete the `append` function so I start typing it, then do `C-n` to select it in the list. It inserts `append(…)` and when I press `c-y` which I have bound to `ncm2_ultisnips#expand_or("\<CR>", 'n')` it expands it with snippets.

https://user-images.githubusercontent.com/5312813/105027542-2e8fed80-5a50-11eb-9a3e-6ee243cbde17.mp4

However, I don't always want to have snippet expansion, for example if I want to add an append function on a line that already exists. Instead, I'd want `c-n` to comlete just `append` and the parens to not be added until I press `c-y`

Is this possible?

I tried setting `rust-analyzer.completion.addCallParenthesis` to false, but that seems to completely disable snippet completion

**Comment by martskins** [comment]
I think that comes as is from `rust-analyzer`, you can verify that if you set the logging level to INFO and check for the response message from the completion call. The `label` on that completion item includes the ellipsis. Might be worth asking in the rust-analyzer repo or zulip, maybe there's a way to have it not do that.
